addon/globalPlugins/TimerForNVDA/conf.py

```diff
@@ -0,0 +1,26 @@
+import config
+
+ADDON_CONFIG = "timerForNVDA"
+
+
+def initConfiguration():
+    confspec = {
+        "reportWithSound": "boolean( default=False)",
+        "reportWithSpeech": "boolean( default=False)",
```

Comment:
Would be interesting if "reportWithSound" or "reportWithSpeech" have a True by default, with both false the user may forget to check some option and the timer will  report nothing.

Reply:
Good point here. Adjjustedto have sound as default reporter. If the user isn't satisfied, the changes they do will be persisted, so this is a good solution.